format-specs/geoparquet.md

```diff
@@ -50,57 +50,100 @@ Version of the geoparquet spec used, currently 0.3.0
+are represented using OGC:CRS84 versus any other geographic CRS.
+
+Note: EPSG:4326 and OGC:CRS84 are equivalent with respect to this specification because this specification specifically overrides the coordinate axis order in the `crs` to be longitude-latitude.
+
+The PROJJSON JSON object for OGC:CRS84 is:
+
+```json
+{
+    "type": "GeographicCRS",
+    "name": "WGS 84 longitude-latitude",
+    "datum": {
+        "type": "GeodeticReferenceFrame",
+        "name": "World Geodetic System 1984",
+        "ellipsoid": {
+            "name": "WGS 84",
+            "semi_major_axis": 6378137,
+            "inverse_flattening": 298.257223563
+        }
+    },
+    "coordinate_system": {
+        "subtype": "ellipsoidal",
+        "axis": [
+        {
+            "name": "Geodetic longitude",
+            "abbreviation": "Lon",
+            "direction": "east",
+            "unit": "degree"
+        },
+        {
+            "name": "Geodetic latitude",
+            "abbreviation": "Lat",
+            "direction": "north",
+            "unit": "degree"
+        }
+        ]
+    },
+    "id": {
+        "authority": "OGC",
+        "code": "CRS84"
+    }
+}
 ```
 
 Due to the large number of CRSes available and the difficulty of implementing all of them, we expect that a number of implementations will start without support for the optional `crs` field.
-Users are recommended to store their data in longitude, latitude (OGC:CRS84 or not including the `crs` field) for it to work with the widest number of tools. But data that is better served in particular projections can choose to use an alternate coordinate reference system. We expect many tools will support alternate CRSes, but encourage users to check to ensure their chosen tool supports their chosen crs.
+Users are recommended to store their data in longitude, latitude (OGC:CRS84 or not including the `crs` field) for it to work with the widest number of tools. Data that are more appropriately represented in particular projections may use an alternate coordinate reference system. We expect many tools will support alternate CRSes, but encourage users to check to ensure their chosen tool supports their chosen CRS.
+
+
+For implementations that operate entirely with longitude, latitude coordinates
+and are not CRS-aware or do not have easy access to CRS-aware libraries that can
+fully parse PROJJSON, it may be possible to infer that coordinates conform to
+the OGC:CRS84 CRS based on elements of the `crs` field.  For simplicity, Javascript
+object dot notation is used to refer to nested elements.
+
+The CRS is likely equivalent to OGC:CRS84 for a GeoParquet file if the `id` element is present:
```

Comment:
Maybe not that important for this PR (can be done in a follow-up as well), but what do people think about moving the details of what the CRS repr of OGC:CRS84 and how to recognize this to a separate section towards the bottom of this file? So here we focus on the basic explanation of this metadata fields